src/sentry/integrations/jira/integration.py

```diff
@@ -692,7 +692,8 @@ def sync_assignee_outbound(self, external_issue, user, assign=True, **kwargs):
                     continue
                 try:
                     jira_user = [
-                        r for r in res if r['emailAddress'] and r['emailAddress'].lower() == ue.email.lower()
+                        r for r in res
+                        if 'emailAddress' in r and r['emailAddress'].lower() == ue.email.lower()
```

Comment:
Any chance of `emailAddress` being present but null? The previous code may have been checking for that.

Maybe `if r.get('emailAddress') and ...`?